src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritMessageProvider.java

```diff
@@ -49,7 +49,7 @@
      * @param build Triggered build to provide custom message for
      * @return the custom message
      */
-    public String getBuildStartedMessage(AbstractBuild build) {
+    public String getBuildStartedMessage(Run build) {
```

Comment:
Will these changes be binary compatible with https://github.com/jenkinsci/build-failure-analyzer-plugin/blob/master/src/main/java/com/sonyericsson/jenkins/plugins/bfa/GerritMessageProviderExtension.java ?